services/src/main/java/org/keycloak/protocol/openshift/OpenShiftTokenReviewEndpoint.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
```

Comment:
it's a picky stuff, but some of the files contain the legal header and some of them do not... Can we unify the approach somehow (my vote is to remove this legal crap... the license in the repo's root).